ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/common-ranGen.yaml

```diff
@@ -6,16 +6,28 @@ metadata:
 spec:
   bindingRules:
     common: "true"
+  remediationAction: inform
   sourceFiles:
     # Create operators policies that will be installed in all clusters
+    - fileName: ClusterVersion.yaml
+      policyName: "cluster-version-policy"
+      remediationAction: inform
+      spec:
+        channel: "4.10"
+        desiredUpdate:
+          force: false
+          version: "4.10"
+          upstream: "upstream"
```

Comment:
We should make test data "generic" and not copy production code in. Having the same content here and in the reference PGT leads to confusion about which set of data is the active code for users.

Reply:
I will update this to generic values